pkg/inspector/image-inspector.go

```diff
@@ -298,3 +322,23 @@ func getAuthConfigs(dockercfg, username, password_file string) (*docker.AuthConf
 
 	return imagePullAuths, nil
 }
+
+func (i *defaultImageInspector) runOpenSCAP() []byte {
```

Comment:
Can this take a Scanner interface rather than creating it itself?  That way we can inject a test scanner and mock up good and bad scans.